x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/Security.java

```diff
@@ -612,6 +619,21 @@ protected Clock getClock() {
         return components;
     }
 
+    protected void possiblySetElasticPassword(SecurityIndexManager.State previousState, SecurityIndexManager.State currentState) {
+        if (previousState.equals(SecurityIndexManager.State.UNRECOVERED_STATE)
```

Comment:
wait until the state of the security index has been recovered from either the disk or a cluster state update.